apps/notifications/src/app/note-list/hooks.ts

```diff
@@ -25,11 +25,8 @@ export function useNoteListFocusToLastSelectedNote( {
 	noteListRef: React.RefObject< HTMLObjectElement >;
 	notes: Note[];
 } ) {
-	const isNoteHidden = useSelector(
-		( state ) => ( noteId: number ) => getIsNoteHidden( state, noteId )
```

Comment:
To resolve the warning because the selector returns a new function every time the component renders